ocw-cli/cli_app.py

```diff
@@ -115,7 +119,7 @@ def load_local_model_screen(header):
                    note = "WARNING: Model file cannot be added."
               elif answer == "1":
                    model_dataset = load_file(model_path, variable_name)
-                   model_datasets.append(model_dataset)
+                   model_datasets.append(dsp.normalize_dataset_datetimes(model_dataset, model_dataset.temporal_resolution()))
```

Comment:
dsp.normalize is important to make every monthly data's day become 1. Otherwise, values of day range from 1 and 30. In many model datasets, day=15 for monthly mean data. This makes users to set up start and end date without confusion. For example, the maximum overlap period is from 1980/01/15 and 2000/12/15. What if a user choose 1980/01/01 and 2000/12/31 as start and end dates respectively?